vendor/bootstrap-datepicker3.css

```diff
@@ -81,6 +81,11 @@
   -ms-user-select: none;
   user-select: none;
 }
+.datepicker-years,
+.datepicker-decades,
+.datepicker-centuries {
+  min-width: 250px;
```

Comment:
Can we make it a bit wider? There should be a little padding (3px, perhaps) between the edge of the blue box and the numbers.

Reply:
Added 30px. Now looks better 
![years](https://user-images.githubusercontent.com/52824207/71520947-56516800-28c7-11ea-8cf8-54b460c48fa7.PNG)
Also I added some min-wdith to months.
![months](https://user-images.githubusercontent.com/52824207/71520949-581b2b80-28c7-11ea-87e9-103ad201f744.PNG)